m1well.Expenses/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Expenses Plugin Changelog
+
+
+## [1.0.0] - 2021-11-24 (@m1well)
+
+### Added
```

Comment:
You don't need to list added files for the first initial commit.  Surely not a show stopper but just food for thought.